lib/metadocs/html_renderer.rb

```diff
@@ -30,15 +30,17 @@ def render_paragraph
     end
 
     def render_table
-      "<table><tbody>#{render_all(element.rows).join}</tbody></table>"
+      "<table><colgroup>#{"<col>" * (element.max_column_count)}</colgroup><tbody>#{render_all(element.rows).join}</tbody></table>"
     end
 
     def render_table_row
       "<tr>#{render_all(element.cells).join}</tr>"
     end
 
     def render_table_cell
```

Comment:
This is context-specific, i.e. only useful for our RTE rather than an assumed behavior for a generic table renderer. So we should move these new behaviors to methods in `PK5Math::HTMLRenderer` or whatever it's called in our project.